InvenTree/InvenTree/version.py

```diff
@@ -9,7 +9,7 @@
 
 import django
 
-from InvenTree.api_version import INVENTREE_API_VERSION
+from .api_version import INVENTREE_API_VERSION
```

Comment:
Why do you decided to import like this? What is the preferred way for imports? Sometimes I see you using relative imports. Sometimes you only  import the direct function, sometimes you import the complete module.

Reply:
Imports in Python are a whole thing. In this specific case, the file will be called both by invoke from the context of / and from other functions in the Django app from the context of /InvenTree. So the absolute import would need to be both InvenTree.InvenTree.api_version and InvenTree.api_version.

We could set the working directory when starting invoke but that will lead to problems once external automation is used (ansible eg.). Therefore I decided that a relative import is the best solution here. In general, I prefer absolute imports for code that should be called from other modules and relative imports when a module is self-reliant and has a defined import point.

Reply:
And what about the second half of my question?

> Sometimes you only import the direct function, sometimes you import the complete module.

Reply:
It is easier to not import 20 different functions and there are some interesting corner cases with Django where it helps to import the module and not a specific function. Over 90% of the time it is the first reason